test/verify/check-storage-luks

```diff
@@ -48,7 +48,7 @@ class TestStorage(StorageCase):
         # Format it with luks
         self.content_row_action(1, "Create Partition")
         self.dialog_wait_open()
-        self.dialog_set_val("size", 10)
+        self.dialog_set_val("size", 17)
```

Comment:
This is a fine workaround for the image refresh. But this will hit real users as well, so we need to handle the situation of "too small" somehow.

Reply:
Agree, I think this should be error message from underlying tooling.